image_tools/src/options.cpp

```diff
@@ -61,7 +61,9 @@ bool parse_command_options(
     ss << "    1 - reliable (default)" << std::endl;
     ss << " -d: Depth of the queue: only honored if used together with 'keep last'. " <<
       "10 (default)" << std::endl;
-    ss << " -f: Publish frequency in Hz. 30 (default)" << std::endl;
+    if( freq != nullptr ){
```

Comment:
I would expect this line to fail the linters. Please run the unit tests of the package locally to check.

Reply:
Got it. Fixed now

Reply:
I don't see any change.

Reply:
I'm very sorry. Now it should be fixed.